### PR TITLE
koordlet: add metrics about be used cpu and node used memory

### PR DESCRIPTION
**Pull request by leason00**
### Ⅰ. Describe what this PR does
metric of newly added BE resource usage.
It has the following functions：
- It is used to sense the proportion of BE resources used
- By comparing with be_suppress_cpu_cores, judge how satisfied the BE Pod is.

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/metrics/common.go [hunk 1]
```diff
@@ -65,6 +65,12 @@ var (
 		Help:      "Number of cpu cores used by node in realtime",
 	}, []string{NodeKey})
 
+	NodeUsedMemory = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: KoordletSubsystem,
+		Name:      "node_used_memory",
```
Comment (saintube) [inline pkg/koordlet/metrics/common.go:70]: `node_used_memory_bytes` is recommended.

File: pkg/koordlet/metricsadvisor/collectors/beresource/be_resource_collector.go [hunk 2]
```diff
@@ -107,7 +108,7 @@ func (b *beResourceCollector) collectBECPUResourceMetric() {
 		klog.Errorf("failed to collect node BECPU, beLimitGenerateSampleErr: %v, beRequestGenerateSampleErr: %v, beUsageGenerateSampleErr: %v", err01, err02, err03)
 		return
 	}
-
+	metrics.RecordBESuppressBEUsedCPU(float64(beCPUUsageMilliCores))
```
Comment (saintube) [inline pkg/koordlet/metricsadvisor/collectors/beresource/be_resource_collector.go:111]: please note that the value's unit is milli-cores

**Comment by leason00** [comment]
@saintube Please review again

File: pkg/koordlet/metrics/common.go [hunk 3]
```diff
@@ -65,6 +65,12 @@ var (
 		Help:      "Number of cpu cores used by node in realtime",
 	}, []string{NodeKey})
 
+	NodeUsedMemory = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: KoordletSubsystem,
+		Name:      "node_used_memory_bytes",
+		Help:      "Memory used by node in realtime",
+	}, []string{NodeKey})
+
 	CommonCollectors = []prometheus.Collector{
```
Comment (saintube) [inline pkg/koordlet/metrics/common.go:74]: need to register the new metric into the `CommonCollectors`

File: pkg/koordlet/metrics/cpu_suppress.go [hunk 4]
```diff
@@ -31,9 +31,16 @@ var (
 		Help:      "Number of cpu cores used by LS. We consider non-BE pods and podMeta-missing pods as LS.",
 	}, []string{NodeKey})
 
+	BESuppressBeUsedCPU = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```
Comment (saintube) [inline pkg/koordlet/metrics/cpu_suppress.go:34]: `BESuppressBeUsedCPU` -> `BESuppressBEUsedCPU `

File: pkg/koordlet/metricsadvisor/collectors/beresource/be_resource_collector.go [hunk 5]
```diff
@@ -107,7 +108,7 @@ func (b *beResourceCollector) collectBECPUResourceMetric() {
 		klog.Errorf("failed to collect node BECPU, beLimitGenerateSampleErr: %v, beRequestGenerateSampleErr: %v, beUsageGenerateSampleErr: %v", err01, err02, err03)
 		return
 	}
-
+	metrics.RecordBESuppressBEUsedCPU(float64(beCPUUsageMilliCores / 1000))
```
Comment (saintube) [inline pkg/koordlet/metricsadvisor/collectors/beresource/be_resource_collector.go:111]: `float64(beCPUUsageMilliCores / 1000)` -> `float64(beCPUUsageMilliCores) / 1000`

**Review by saintube — APPROVED** [review]
/lgtm

**Pull request merged**